refs/notion-of-privacy/notion-of-privacy.md

```diff
@@ -57,39 +59,45 @@ Compelling animals to remain in contact contrary to their own privacy inclinatio
 
 Beyond the privacy of an individual, privacy also has a group-preserving function in the relationship between one group to another[^15].
 
-## Privacy and Other Topics
+### For Humans, Connection = Survival
 
-### Privacy and Information Asymmetry
+Humans are social species, hardwired for connection.
 
-Information asymmetry[^8] is clearly a key concept for privacy as identified by biological studies of privacy in animal societies.
+> « **Connection is the energy that exists between people when they feel seen, heard and valued; when they can give and receive without judgement; and when they derive sustenance and strength from the relationship.** » &mdash; _Brené Brown_
```

Comment:
This may just be due to my own mindset and background.
I learned computer networking and system engineering before communication science, and communication science via cybernetics. Even now, I tend to see communication between social entities through this lens.

So when I read "connection", I first think about "channels", as in the Shannon–Weaver model or [Telecommunication](https://en.wikipedia.org/wiki/Telecommunications_link). Models where psychological, cultural and social aspects of communication tend to be considered marginally, as "noise".

Here, Brown's definition focus on interpersonal connections, with a strong emphasis on feelings and social/psychological aspects ("when they feel seen, heard and valued", "without judgement", "relationship"). In the context of this specific document, this is perfectly acceptable, as we can derive this meaning from the context (and starting with this quote is indeed a really good way to clarify this directly).

However, I expect this vocabulary to be soon used extensively in our future organizational and promotional communications. And I also expect third-party developers to have the same biases as I have.

I'd like to keep this discussion going so we can define a vocabulary allowing our audience to understand everything we share without us needing the repeat a definition each time.

Reply:
I created an associated issue so we can discuss this further without blocking this PR.

Reply:
I agree with you. 

We can further improve clarity by stating that we are not refering to the notion of `connection` as a [network connection](https://en.wikipedia.org/wiki/Telecommunications_link).

Maybe a good idea is to stick with Brown's definition of connection, but then in the rest of the text, focus on connectitdness ( the state of being in connection ) as use that term to avoid clash with overly common connection.

However, this is a longer discussion, and I agree to continue it in https://github.com/blindnet-io/communication-management/issues/127 and finish this here for now.

Reply:
I like connectidness :+1:

Anyway, I clarified my expectations in https://github.com/blindnet-io/communication-management/issues/127#issuecomment-1137028418. I don't expect this to have any consequence on this text which is really good as it is (being "context (1)"). Resolving the conversation.